draft-jholland-quic-multicast.md

```diff
@@ -340,9 +344,9 @@ MC_CHANNEL_PROPERTIES frames contain the following fields:
   * Max Idle Time: The maximum expected idle time of the channel.  If this amount of time passes in a joined channel without data received, clients SHOULD leave the channel with reason Max Idle Time Exceeded.
   * ACK Bundle Size: The minimum number of ACKs a client should send in a single QUIC packet. If the max_ack_delay would force a client to send a packet that only consists of MC_CHANNEL_ACK frames, it SHOULD instead wait with sending until at least the specified number of acknowledgements have been collected. However, the Client MUST send any pending acknowledgements at least once per Max Idle Time to prevent the Server from perceiving the channel as interrupted.
 
-From Packet Number and Until Packet Number are used to indicate the packet number (Section 17.1 of {{RFC9000}}) the 1-RTT packets received over which these values are applicable.
+From Packet Number and Until Packet Number are used to indicate the packet numbers (Section 17.1 of {{RFC9000}}) of the 1-RTT packets received for which the values contained in a MC_CHANNELS_PROPERTIES frame are applicable.
 
-A From Packet Number without an Until Packet Number has an unspecified termination.
+If a frame contains a From Packet Number without an Until Packet Number, there is no specified termination to the applicability of its values.
```

Comment:
There is always now an Until Packet Number, but I think the value 0 could be used here to indicate indefinite termination.